src/ensembl/src/content/app/browser/hooks/useBrowserRouting.ts

```diff
@@ -141,9 +140,9 @@ const useBrowserRouting = () => {
         location: chrLocation ? getChrLocationStr(chrLocation) : null
       };
 
-      dispatch(resetBrowserTrackStates());
-
-      dispatch(replace(urlFor.browser(params)));
+      batch(() => {
+        dispatch(replace(urlFor.browser(params)));
+      });
```

Comment:
Couple of things about batch:
1) it only makes sense with multiple dispatches in a row
2) even then, according to the docs, it only makes sense outside of React ("You can use it to ensure that multiple actions dispatched **outside of React** only result in a single render update", [link](https://react-redux.js.org/api/batch))

Bottom line: no need for the batch here.

Reply:
Yes, I just removed one dispatch but forgot to remove batch. That's done now

Reply:
Umm, it's still in the diff 😅

Reply:
Just pushed it. Sorry!